packages/rake-db/src/migration/migrationList.ts

```diff
@@ -0,0 +1,31 @@
+import { ColumnTypesBase } from 'orchid-core';
+import { DefaultColumnTypes } from 'pqb';
+import { RakeDbConfig, getMigrations } from 'src/common';
+
+const getFileNameFromPath = (path: string) => {
+  const pathArray = path.split('/');
```

Comment:
Windows turns slashes incorrectly, so that won't work on windows.
Use `path.filename` instead from 'node:path'.